Jenkinsfile

```diff
@@ -0,0 +1,92 @@
+// To use a test branch (i.e. PR) until it lands to master
+// I.e. for testing library changes
+// Phyl : from environment
+//        TR_REDIRECT_OUTPUT = 'yes'
+// Phyl : from test stage
+//                                export TR_REDIRECT_OUTPUT='yes'
+@Library(value="pipeline-lib@test-branch") _

```

Comment:
FWIW, this line really goes with the comments on line [1](https://github.com/daos-stack/cart/pull/8/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R1) and [2](https://github.com/daos-stack/cart/pull/8/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R2).

But more importantly, unless you are testing an update to the library (i.e. in a PR), you should have this line commented out so that your pipeline is using whichever "production" branch of the library that the Jenkins organisation has configured.  I will be changing that configured branch shortly and you don't want to be stuck following the `test-branch` branch when I do.

```suggestion
//@Library(value="pipeline-lib@test-branch") _
```